### ISSUE TITLE
Suggestion: add Lambert W x F theoretical distributions in descdist()

### ISSUE THREAD

**Issue by gmgeorg** [aursiber/fitdistrplus#25]
Suggest to add an option to show Lambert W x F options in the "theoretical distribution" in the Cullen and Frey  graph (`descdist`).

**Tl;dr**: Lambert W x F generalize any F distribution to be (more) skewed and/or heavy tailed.  See [LambertW](https://github.com/gmgeorg/LambertW) R package documentation and references therein for details. It thus lends itself naturally to fill up the areas of the Cullen and Frey graph in areas where the standard distributions do not have coverage 

In particular 
* a line along (skewness > 0, kurtosis > 3) for a skewed Lambert W x Gaussian distribution (see Eq. (4.3) & (4.4) in [Goerg 2011](https://www.jstor.org/stable/23069370))
* a vertical line at skew = 0 for kurtosis >=3 for heavy tailed Lambert W x normal distributions ([Goerg 2015](https://www.hindawi.com/journals/tswj/2015/909231/))
* a skew and kurtosis curve for Lambert W x Exponential distributions ([Goerg 2011](https://www.jstor.org/stable/23069370) and [Kaartik 2023](https://www.mdpi.com/2073-8994/15/10/1877)) starting from the (4,9) point of the "Exponential" distribution towards (inf, inf) ( a Lambert W x Exponential reduces to an exponential for $\gamma=0$). See chart attached for the theoretical values of (skew^2, kurtosis) as a function of gamma>=0.
* a lower triangle area for a double heavy tail Lambert W x Gaussian distribution (see chart for empirical estimates) 

Theoretical skewness and kurtosis as a function of $\gamma$ for Lambert W x Normal (see Goerg 2011 for derivations)
![9faa82b0-55ce-4a9a-bc82-2b01cc54d542](https://github.com/aursiber/fitdistrplus/assets/48655643/6daef56f-3bbb-4b64-98f3-eb230ca57c8e)

Theoretical skewness and kurtosis as a function of $\gamma$ for Lambert W x Exponential (see Kaartik for derivations)
![892f2d65-a355-4e52-971e-d4022ab114ec](https://github.com/aursiber/fitdistrplus/assets/48655643/0cd74a6b-f645-4559-adb8-4600490f7e5e)

Empirical estimates for the "lower triangle" area of double-tail Lambert W x GAussian distriubtions (including the vertical line at (skew = 0, kurtosis >= 3 -- which holds in theory for heavy-tail Lambert W x Gaussian distribution and $\delta >= 0$)

![e5569175-9f62-4e07-8cb2-1ca071a1953b](https://github.com/aursiber/fitdistrplus/assets/48655643/3df39b23-3734-4317-a0b4-eb83d97db9b3)

To replicate empirical simulation of double-tail Lambert W x Gaussian you can use

```r
library(LambertW)

delta_l <- seq(0, 0.3, by=0.01)
delta_r <- seq(0, 0.3, by=0.01)
set.seed(10)

res = list()
counter = 0
for (dl in delta_l) {
  
  for (dr in delta_r) {
    counter = counter + 1
    x <- rLambertW(n=100000, "normal", theta=list(beta=c(mean=0, sd = 1), 
                                                    delta=c(dl, dr)))

    res[[counter]] = c("delta_l" = dl, "delta_r" = dr,
                       "mean" = mean(x), 
                       "var" = var(x), "skewness" = skewness(x), 
                       "kurtosis" = kurtosis(x))
  }
}

res.df.hh = as.data.frame(do.call(rbind, res))

plot(res.df.hh$skewness**2,
     res.df.hh$kurtosis,
     xlab="square of skewness", ylab="kurtosis",
     main="Lambert W x Normal",
     xlim=c(0, 15),
     ylim=c(40, 1),
     pch=19)

mask = res.df.hh$delta_l == res.df.hh$delta_r
points(res.df.hh[mask,]$skewness**2,
       res.df.hh[mask,]$kurtosis,
       pch=19, col="orange")
points(0, 3, pch=19, col="red")
grid()

legend("topright", 
       c("Normal",
         "heavy-tail Lambert W x Normal",
         "double-tail Lambert W x Normal"),
       col=c("red", "orange", "black"), pch = 19)

```

**Comment by dutangc** [comment]
Thanks for the idea. Added to our todolist.
